resources/assets/uad_lists.json

```
@@ -13440,6 +13472,14 @@
     "labels": [],
     "removal": "Advanced"
   },
+  "com.mediatek.videoplayer": {
+    "list": "Misc",
+    "description": "The com.mediatek.videoplayer package is MediaTek’s default video player app, designed to handle video playback and provide basic media management features. Since there are numerous alternative video player apps available on Google Play that offer advanced features and better performance, removing this package is generally recommended.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```

Native apps sometimes tied to OS.